src/pybind/mgr/cephadm/module.py

```diff
@@ -2272,7 +2302,7 @@ def _create_daemon(self, daemon_type, daemon_id, host,
             if daemon_type == 'osd':
                 if not osd_uuid_map:
                     osd_uuid_map = self.get_osd_uuid_map()
-                osd_uuid = osd_uuid_map.get(daemon_id, None)
+                osd_uuid = osd_uuid_map.get(daemon_id)
```

Comment:
(this will now raise a `KeyError`, if it is missing)

Reply:
no, see https://github.com/ceph/ceph/pull/34346#discussion_r402281397